### DISCUSSION TITLE
Install complement in Office 365 Administration Center

### DISCUSSION THREAD

**Discussion by akrog79** [0dteam/Phishing-Reporter#9] [Ideas]
Hello,

First of all, thanks to share your project.

I need to install the add-in using Office 365 Administration Center. How can I add it? Because I need a .xml file but the installer is a .exe file.

Thanks!

**Comment by 0dteam** [comment]
Hello akrog79,
Phishing-Reporter is a VSTO add-in; which you can push using SCCM or group policies, but not O365 Administration Center.
https://social.msdn.microsoft.com/Forums/office/en-US/c9675499-bd0a-4bc9-a894-7713f1f63ceb/can-vsto-addin-be-deployed-via-office-365-admin-center?forum=worddev

I might release an (xml-based) add-in for Office 365 soon. If I do, I will make sure to tag you here with the link.

**Reply by akrog79** [reply]
Thanks a lot!

It's a nice idea to release an xml-based add-in.

I understand so that this add-in doesn't works for Outlook Office 365 Desktop Client, right?

**Comment by szvvo** [comment]
Hello there, any news about releasing the XML-based add in?
